aquadoggo/src/bin/dump_gql_schema.rs

```diff
@@ -0,0 +1,12 @@
+use aquadoggo::db::connection_pool;
+use aquadoggo::graphql::{build_root_schema, Context};
+
+#[tokio::main]
+async fn main() {
```

Comment:
If we add a `default-members = ["aquadoggo_cli"]` in the workspace `Cargo.toml` one can still run `cargo run` and it will start the aquadoggo node. Now with two binary targets one receives an error which makes sense, but it would be nice to easily make it work with a default